evennia/utils/dbserialize.py

```diff
@@ -153,6 +153,12 @@ def _save_tree(self):
         if self._parent:
             self._parent._save_tree()
         elif self._db_obj:
+            if not self._db_obj.pk:
+                cls_name = self.__class__.__name__
+                non_saver_name = cls_name.lstrip("_Saver")
```

Comment:
It's a small thing so I'll just fix this before I merge, but if you are not aware, this is *not* the way to strip a fixed string from the beginning of another string. What `lstrip` (and `rstrip` as well as `strip`) takes for argument is a *sequence of characters* to remove. The order of them does not matter, nor how many that should be removed. To see what I mean, try what you get if you do 
```python
"testtsettttsseset".lstrip("test")
```

Unless you want to use indexing (`cls_name[5:]`) I think the commonly recommended to strip an initial string is to use split: 
```python
non_saver_name = cls_name.split("_Saver", 1)[1]
```